runtime-modules/content/src/lib.rs

```diff
@@ -129,6 +132,9 @@ decl_storage! {
 
         pub NextVideoId get(fn next_video_id) config(): T::VideoId;
 
+        /// Auction Nonce
+        pub NextAuctionId get(fn next_auction_id) config(): T::AuctionId;
```

Comment:
First off, we don't actually need ids for all auctions, we need unique nonces specifically for _open auctions_. So at the very least, a renaming is needed here, but I actually suggest below - related to other more complex type issues about `AuctionRecord`, to just have runtime use block height as nonce. Its not a problem that different auctions have same nonce, we only need a unique nonce _per video_.

You can hypotheically imagine a weird scenario where an auction is opened, a bid is made, a new auction is opened, and then the old bid is accepted for the new auction, all in the same block! 🤯 I find this highly highly unlikely ot be possible.